language-server/modules/langserver-core/src/test/resources/codeaction/getter-setter/source/getterSetter5.bal

```diff
@@ -0,0 +1,13 @@
+class Person {
+
+    private int age   ;
+
+    public function getAge() returns int {
+        return self.age;
+    }
+
+    public function setAge(int age) {
+        self.age = age;
+    }
+
+}
```

Comment:
```suggestion
}

```

Reply:
Updated in 346328f74b6b1c25f5a6e0a93966e27bd829d265